test/open_pgp/literal_data_packet_test.exs

```diff
@@ -1,4 +1,43 @@
 defmodule OpenPGP.LiteralDataPacketTest do
   use OpenPGP.Test.Case, async: true
   doctest OpenPGP.LiteralDataPacket
+  doctest OpenPGP.Encode.impl_for!(%OpenPGP.LiteralDataPacket{})
+
+  alias OpenPGP.Encode
+  alias OpenPGP.LiteralDataPacket
+
+  describe "OpenPGP.Encode.encode/1,2" do
+    test "encode plaintext with default opts" do
+      packet = %LiteralDataPacket{data: "Hello"}
+      assert <<0x62, 0, unix_ts::32, "Hello">> = Encode.encode(packet)
+      assert_in_delta unix_ts, System.os_time(:second), 2
+    end
+
+    test "encode plaintext and set file name" do
+      packet = %LiteralDataPacket{file_name: "file.txt", data: "Hello"}
+      assert <<0x62, 8, "file.txt", _::32, "Hello">> = Encode.encode(packet)
```

Comment:
Any way we can assert the timestamp here as well similar to the other tests?

Reply:
I guess I could, but I've tested it in the other test above (the default timestamp is current ts). This one is focused on file name octets. Are you OK keeping those separate?